backend/package.json

```diff
@@ -4,12 +4,17 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest",
+    "lint": "./node_modules/.bin/eslint"
```

Comment:
Make this `lint` script use `eslint` instead. It should run `eslint` on all `.js` files under the `src` directory

https://stackoverflow.com/questions/36307581/how-to-configure-package-json-to-run-eslint-script